charts/posthog/values.yaml

```diff
@@ -39,12 +39,10 @@ env: []
 #   - name: FOO
 #     value: bar
 
-
```

Comment:
Lots of formatting changes were committed unrelated to this change? Not sure if this was expected.

Reply:
It was. My IDE automatically formats yaml. Seems like most of this makes sense to me.